core/src/main/java/cz/o2/proxima/scheme/ValueSerializer.java

```diff
@@ -110,4 +110,14 @@ default String getLogString(T value) {
     throw new UnsupportedOperationException(
         getClass() + " is not ported to provide a ValueSchemaDescriptor. " + "Please fill issue.");
   }
+
+  /**
+   * Provide {@link AttributeValueAccessor} for give attribute value.
+   *
+   * @return value accessor
+   */
+  default <V> AttributeValueAccessor<T, V> getValueAccessor() {
+    throw new UnsupportedOperationException(
+        getClass() + " is not ported to provide a ValueAccessor. " + "Please fill issue.");
```

Comment:
nit: Probably no need to separate the two strings. But can be fixed by different PR (I think I already did that somewhere).

Reply:
nit: same applies to schema descriptor above

Reply:
applied on all.